assessor/bin/assess-minibroker-charts.rb

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env ruby
+##
+# The purpose of this script is to assess which of the publicly
+# available charts for the databases supported by minibroker will work
+# with SCF.
+
+# Configuration
+# - Fixed location: `stable` helm repository
+# - Configurable:   work directory for state
+# - Configurable:   scf source directory
+# - Configurable:   scf namespace
+# - Configurable:   Cluster Admin Password
+# - Configurable:   Operation mode (full, incremental)
+
+require 'fileutils'
+require 'net/http'
+require 'open3'
+require 'optparse'
+require 'uri'
+require 'yaml'
+
+require_relative "../lib/colors.rb"
+require_relative "../lib/config.rb"
+require_relative "../lib/helmrepo.rb"
+require_relative "../lib/locations.rb"
+require_relative "../lib/log.rb"
+require_relative "../lib/spawn.rb"
+require_relative "../lib/state.rb"
+require_relative "../lib/terminal.rb"
+require_relative "../lib/testcore.rb"
+
+@top = File.dirname(File.dirname(File.dirname(File.absolute_path(__FILE__))))
+
+def main
+  config
+  base_statistics
+
+  @assessed = 0
+  @skipped = 0
+  state
```

Comment:
Oops, missed this one… we don't need to explicitly call it; just use it as an accessor. (That is: instead of using `@state` explicitly elsewhere, just use `state` like a variable.)

Reply:
will remove

Reply:
Addressed with commit b671d3d1eddb106ceee500f0c59ad31eb3b0cba1